translator.py

```diff
@@ -273,6 +275,11 @@ def cli():
                         default=None,
                         help='Token of cqhttp, if it is not set on the server side, '
                         'it does not need to fill in.')
+    parser.add_argument('--gpt_base_url',
+                        type=str,
+                        default="https://api.openai.com/v1/",
```

Comment:
The official API URL may also change, I prefer `default=None` here.

Reply:
> The official API URL may also change, I prefer `default=None` here.

Okey